include/swift/AST/Attr.h

```diff
@@ -1595,7 +1595,7 @@ class DifferentiableAttr final
 
   // Print the attribute to the given stream.
   void print(llvm::raw_ostream &OS, const Decl *D,
-             ModuleDecl *prettyPrintInModule = nullptr) const;
+             bool omitDifferentiationParametersClause = false) const;
```

Comment:
`omitWRTClause` is shorter and more directly relatable.

Reply:
How about `omitWrtClause`?

Reply:
Renamed to `omitWrtClause` in https://github.com/apple/swift/pull/25324/commits/12af26bb99bf20544df43cbc6a78e026c697ca05.